test_scripts/API/AutoCompleteList/016_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_not_Respond_GENERIC_ERROR.lua

```diff
@@ -0,0 +1,59 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = false, resultCode = "GENERIC_ERROR"> to mobile app
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList> 
+-- 3. HMI does NOT respond during <DefaultTimeout>
```

Comment:
@Aleksandriys, looks like you missed "Expected result"(mandatory in template) in header.
I guess, at least p3. and p4 should be in expected result.